crates/matrix-sdk/tests/integration/room/timeline.rs

```diff
@@ -415,8 +406,8 @@ async fn reaction() {
         timeline_stream.next().await,
         Some(VecDiff::UpdateAt { index: 1, value }) => value
     );
-    let event_item = updated_message.as_event().unwrap();
-    let msg = assert_matches!(event_item.content(), TimelineItemContent::Message(msg) => msg);
+    let event_item = updated_message.as_event().unwrap().as_remote().unwrap();
+    let msg = assert_matches!(event_item.content, TimelineItemContent::Message(ref msg) => msg);
```

Comment:
Let's not use `ref` please.

```suggestion
    let msg = assert_matches!(&event_item.content, TimelineItemContent::Message(msg) => msg);
```